google-hangout-button.html

```diff
@@ -40,62 +44,84 @@
      * @param {Object} e Event parameters.
      */
 
-    /**
-     * Specifies what type of Hangout should be started.
-     * Valid values are 'normal', 'onair', 'party' and 'moderated'
-     *
-     * See the [Hangout button parameter reference](https://developers.google.com/+/hangouts/button#hangout_button_parameters)
-     * for more details.
-     *
-     * @attribute type
-     * @type string
-     * @default 'normal'
-     */
-    type: 'normal',
+     properties: {
 
-    /**
-     * Specifies the Google+ Hangout apps to launch when a user clicks the
-     * Hangout button. Invalid objects and parameters are ignored.
-     *
-     * See the [Initial app parameters reference](https://developers.google.com/+/hangouts/button#initial_app_parameters)
-     * for more details.
-     *
-     * @attribute apps
-     * @type Array.<object>
-     */
-    apps: null,
+      /**
+       * Specifies what type of Hangout should be started.
+       * Valid values are 'normal', 'onair', 'party' and 'moderated'
+       *
+       * See the [Hangout button parameter reference](https://developers.google.com/+/hangouts/button#hangout_button_parameters)
+       * for more details.
+       *
+       * @attribute type
+       * @type string
+       * @default 'normal'
+       */
+      type: {
+        type: String,
+        value: 'normal'
+      },
 
-    /**
-     * Specifies the list of people to invite when the user clicks the
-     * Hangout button. Invalid objects and parameters are ignored.
-     *
-     * See the [Invite parameters reference](https://developers.google.com/+/hangouts/button#invite_parameters)
-     * for more details.
-     *
-     * @attribute invites
-     * @type Array.<object>
-     */
-    invites: null,
+      /**
+       * Specifies the Google+ Hangout apps to launch when a user clicks the
+       * Hangout button. Invalid objects and parameters are ignored.
+       *
+       * See the [Initial app parameters reference](https://developers.google.com/+/hangouts/button#initial_app_parameters)
+       * for more details.
+       *
+       * @attribute apps
+       * @type Array.<object>
+       */
+      apps: {
+        type: Array,
+        value: []
```

Comment:
@ebidel qq for my own interest: is there a reason why a static `[]` couldn't be used here? In the other elements I've seen use a function for the `value` it's been because the value required computation or value lookup from elsewhere.